### PR TITLE
Feat/15496317 members page

### PR DESCRIPTION
**Pull request by CzarekDryl**
Added members pages, routing and subnavigation with members items

https://tw.auroracreation.com/app/tasks/15496317
https://tw.auroracreation.com/app/tasks/15496319
https://tw.auroracreation.com/app/tasks/15496366

### CODE REVIEW

File: src/components/common/Extensions/MainNavigation/partials/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { useMemo } from 'react';
+import { NavItem } from './types';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/MainNavigation/partials/hooks.ts:2]: Add a blank space between the imports - I think import packages and other imports should be separated by the blank line for better readability.

File: src/components/frame/Extensions/layouts/PageLayout.tsx [hunk 2]
```diff
@@ -58,8 +64,11 @@ const ExtensionsLayout: FC<PropsWithChildren> = ({ children }) => {
     applyTheme(isDarkMode ? Theme.dark : Theme.light);
   }, [isDarkMode]);
 
+  // @TODO: Handle other pages when needed or handle navigation in the page itself
+  const pageName = loadingText === 'extensionPage' ? 'extensions' : 'members';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/layouts/PageLayout.tsx:68]: Shouldn't be a translation here?

File: src/components/frame/Extensions/layouts/PageLayout.tsx [hunk 3]
```diff
@@ -87,11 +96,11 @@ const ExtensionsLayout: FC<PropsWithChildren> = ({ children }) => {
       </div>
       <main className="mt-9 pb-24">
         <div className="inner">
-          {isMobile && <Navigation />}
+          {isMobile && <Navigation pageName={pageName} />}
           <div className="mt-9 sm:mt-6">
             <PageTitle
-              title={formatMessage({ id: 'extensionsPage.title' })}
-              subtitle={formatMessage({ id: 'extensionsPage.description' })}
+              title={formatMessage(title)}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/layouts/PageLayout.tsx:102]: Maybe would be better to move formatMessage to the PageTitle component?

File: src/components/frame/v5/pages/ContributorsPage/ContributorsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from 'react';
+import Navigation from '~v5/common/Navigation/Navigation';
+import TwoColumns from '~v5/frame/TwoColumns/TwoColumns';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/v5/pages/ContributorsPage/ContributorsPage.tsx:3]:
```
import React, { FC } from 'react';

import Navigation from '~v5/common/Navigation';
import TwoColumns from '~v5/frame/TwoColumns';
```

File: src/components/frame/v5/pages/FollowersPage/FollowersPage.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from 'react';
+import Navigation from '~v5/common/Navigation/Navigation';
+import TwoColumns from '~v5/frame/TwoColumns/TwoColumns';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/v5/pages/FollowersPage/FollowersPage.tsx:3]:
```
import React, { FC } from 'react';

import Navigation from '~v5/common/Navigation/Navigation';
import TwoColumns from '~v5/frame/TwoColumns/TwoColumns';
```

Comment (KatarzynaStefanowiczPagepro) [reply src/components/frame/v5/pages/FollowersPage/FollowersPage.tsx:3]: Please, check it and change everywhere

File: src/routes/Routes.tsx [hunk 6]
```diff
@@ -69,6 +72,11 @@ import AdvancedPage from '~frame/Extensions/pages/AdvancedPage';
 import PermissionsPage from '~frame/Extensions/pages/PermissionsPage';
 import LazyConsensusPage from '~frame/Extensions/pages/LazyConsensusPage';
 import { ExtensionsContextProvider } from '~context/ExtensionsContext';
+import MembersPage from '~frame/v5/pages/MembersPage/MembersPage';
+import ContributorsPage from '~frame/v5/pages/ContributorsPage/ContributorsPage';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/routes/Routes.tsx:76]: `import ColonyDetailsPage from '~frame/Extensions/pages/ColonyDetailsPage';`

Please, change in other places and check your settings in VSCode to import the component

File: src/components/frame/Extensions/layouts/PageLayout.tsx (hunk 2 above)
Comment (CzarekDryl) [reply src/components/frame/Extensions/layouts/PageLayout.tsx:68]: It's only used as id, so we don't need translation here

File: src/components/common/Extensions/MainNavigation/partials/hooks.ts (hunk 1 above)
Comment (CzarekDryl) [reply src/components/common/Extensions/MainNavigation/partials/hooks.ts:3]: Fixed

File: src/components/frame/Extensions/layouts/PageLayout.tsx (hunk 3 above)
Comment (CzarekDryl) [reply src/components/frame/Extensions/layouts/PageLayout.tsx]: moved to pageTitle

File: src/components/frame/v5/pages/ContributorsPage/ContributorsPage.tsx (hunk 4 above)
Comment (CzarekDryl) [reply src/components/frame/v5/pages/ContributorsPage/ContributorsPage.tsx]: Fixed

File: src/components/frame/v5/pages/FollowersPage/FollowersPage.tsx (hunk 5 above)
Comment (CzarekDryl) [reply src/components/frame/v5/pages/FollowersPage/FollowersPage.tsx]: Fixed

File: src/routes/Routes.tsx (hunk 6 above)
Comment (CzarekDryl) [reply src/routes/Routes.tsx]: Fixed

File: src/routes/Routes.tsx [hunk 7]
```diff
@@ -58,7 +61,7 @@ import NotFoundRoute from './NotFoundRoute';
 import { ColonyContextProvider } from '~context/ColonyContext';
 import CreateColonyWizard from '~common/CreateColonyWizard';
 import ActionDetailsPage from '~common/ColonyActions/ActionDetailsPage';
-import ExtensionsLayout from '~frame/Extensions/layouts/ExtensionsLayout';
+import PageLayout from '~frame/Extensions/layouts/PageLayout';
 import ExtensionDetailsPage from '~frame/Extensions/pages/ExtensionDetailsPage';
 import ColonyDetailsPage from '~frame/Extensions/pages/ColonyDetailsPage/ColonyDetailsPage';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/routes/Routes.tsx:66]: `import ColonyDetailsPage from '~frame/Extensions/pages/ColonyDetailsPage';`

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**